changelog.md

```diff
@@ -6,6 +6,8 @@
 - bpk-component-button:
   - Fixed SyntheticEvent not being passed to the onClick handler.
 
+## 2017-08-23 - added 1 new icon (unlock)
```

Comment:
As this has not yet been released, please place the following under the `unreleased > added` heading

```
- bpk-component-icon:
  - Added 1 new icon (unlock)
```